geoportal/c2cgeoportal_geoportal/views/login.py

```diff
@@ -324,17 +320,16 @@ def change_password(self) -> pyramid.response.Response:
         else:
             user = self.request.user
 
-        username = self.request.registry.validate_user(self.request, user.username, old_password)
-        if username is None:
-            LOG.info("The old password is wrong for user '%s'.", username)
+        if self.request.registry.validate_user(self.request, user.username, old_password) is None:
+            LOG.info("The old password is wrong for user '%s'.", user.username)
```

Comment:
Shouldn't we return standard response here for security.
=> do not give the hacker the information that the username exists

Reply:
The visible response is on the next line and is completely generic :-)
Or I miss something...

Reply:
Right sorry